core/tests/release_sources/valid_release_mail_message.txt

```diff
@@ -9,4 +9,4 @@ Thanks to Bob and Paul, our returning contributors who made this release possibl
 Finally, I'd like to thank Smith for their help with pre-release testing, bug-fixing and QA, as well as Tom for leading the QA team for this release.
 
 Thanks,
-Ruth
\ No newline at end of file
+Ruth
```

Comment:
What is the change here?

Reply:
Newline is added after Ruth. Now it is: `Ruth\n`